test/google_spreadsheet_test.exs

```diff
@@ -4,12 +4,46 @@ defmodule GoogleSpreadsheetTest do
 
   @tag :normal
   test "send_new_message are pushed to the client" do
-    # spreadsheet_id = "11-uGmVZiO8ZZ6MWNAf4pKsgrmuW5cMNxpSMm7VY9-u0"
-    # IO.inspect(spreadsheet_id, label: "spreadsheet_id")
+    spreadsheet_id = "11-uGmVZiO8ZZ6MWNAf4pKsgrmuW5cMNxpSMm7VY9-u0"
```

Comment:
is this a real spreadsheet? if so we need to make it public to anyone as this repo is public and other people might want to run tests